tests/test-cli-extensions.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2021 Red Hat Inc.
+# SPDX-License-Identifier: LGPL-2.0+
+
+set -euo pipefail
+
+. $(dirname $0)/libtest.sh
+
+echo '1..2'
+
+mkdir -p ./localbin
+export PATH="./localbin/:${PATH}"
+ln -s /usr/bin/env ./localbin/ostree-env
+${CMD_PREFIX} ostree env --help >out.txt
+assert_file_has_content out.txt "with an empty environment"
```

Comment:
How about e.g.:
```suggestion
${CMD_PREFIX} env sometestenv=foo ostree env >out.txt
assert_file_has_content out.txt '^sometestenv=foo`
```

or so?

Reply:
I also realized there were some leftovers in the environment. https://github.com/ostreedev/ostree/pull/2501 takes care of all that.